src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -5124,6 +5124,33 @@ static void Main()
         }
 
         [Fact]
+        public void SingleDiscardInAssignmentInCSharp6()
+        {
+            var source =
+@"
+class C
+{
+    static void Error()
+    {
+        _ = 1;
+    }
+    static void Ok()
+    {
+        int _;
+        _ = 1;
+        System.Console.Write(_);
+    }
+}
+";
+            var comp = CreateCompilationWithMscorlib(source, parseOptions: TestOptions.Regular6);
+            comp.VerifyDiagnostics(
+                // (6,9): error CS8059: Feature 'tuples' is not available in C# 6.  Please use language version 7 or greater.
+                //         _ = M();
+                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion6, "_").WithArguments("tuples", "7").WithLocation(6, 9)
```

Comment:
I didn't add explicit test for discard in other scenarios, as they involve new syntax (out var, pattern matching, deconstruction) and we already have tests for such new syntax with langver=6. I don't mind adding one though.